src/core/AutoRest.Core/Properties/Resources.resx

```diff
@@ -241,6 +241,9 @@ Modelers:
   <data name="TrackedResourceListByImmediateParentOperationMissing" xml:space="preserve">
     <value>The child tracked resource, '{0}' with immediate parent '{1}', must have a list by immediate parent operation.</value>
   </data>
+  <data name="XMSPagableListByRGAndSubscriptionsMismatch" xml:space="preserve">
+    <value>For the tracked resource '{0}', the x-ms-pageable must be same for list by resource group and subscriptions operations.</value>
```

Comment:
Suggestion for message: "List by resource group and list by subscription operations for tracked resource `{0}` should be consistent for paging, either both should have "x-ms-pageable" extension, or both shouldn't.